javascript/example-customer-account--loyalty-app--ts/extensions/customer-account-loyalty-fulfillment-details-extension/README.md

```diff
@@ -0,0 +1,3 @@
+# Customer account UI Extension
```

Comment:
You can replace the content here with this one https://github.com/Shopify/customer-account-tutorials/blob/main/react/example-customer-account--loyalty-app--react/README.md